src/util/warcwriter.ts

```diff
@@ -60,8 +60,6 @@ export class WARCWriter implements IndexerOffsetLength {
   }) {
     this.archivesDir = archivesDir;
     this.tempCdxDir = tempCdxDir;
-    // for now, disabling CDX
-    this.tempCdxDir = undefined;
```

Comment:
One thing to add regarding tempCdxDir - it's now required to be kept so that we don't have to reindex the WARCs again, if adding to an existing collection. Perhaps should rename to 'warcCdx' or some other name to indicate its not as temporary, eg. it'll be needed again if adding to collection or regenerating WACZ again, so we don't have to reindex.

Reply:
We could regenerate the CDX from WARCs, but no reason to delete them as they're pretty small compared to the WARCs, so just keeping them around in the collection

Reply:
Good point! I'm in favor of renaming it, and might be good to document what this is and when you might need it in the crawler docs.

Reply:
Renamed to `warcCdxDir` and using `warc-cdx` instead of `tmp-cdx` for the dir name, in https://github.com/webrecorder/browsertrix-crawler/pull/673/commits/fb427eae146a84e89d8451c037dd5f0ba9464b4f

Reply:
That makes sense. Only thing is I think if I didn't have the "dev on the team" perspective, I might be confused why there are indexes in both `warc-cdx` and `indexes` (assuming `--generateCDX` or `--generateWACZ` was used), and how they differ, so would be good to add something to the documentation about this before merging.

Reply:
Created new issue for that so as not to block this PR: https://github.com/webrecorder/browsertrix-crawler/issues/675